api/libproject/devstart.rb

```diff
@@ -634,6 +634,50 @@ def run_local_all_migrations()
   :fn => ->() { run_local_all_migrations() }
 })
 
+def rollback_db(cmd_name, *args)
+  puts "in rollback_db. cmd_name: #{cmd_name} args: #{args}"
+  ensure_docker_sync
+  op = WbOptionsParser.new(cmd_name, args)
+  op.add_typed_option(
+      "--count=[count]",
+      Integer,
+      ->(opts, c) { opts.count = c},
+      "Count of changesets to roll back from end.")
+
+  op.add_typed_option(
+      "--tag [tag]",
```

Comment:
Perhaps a more specific flag name, e.g. --to-tag ; currently it's slightly ambiguous as to whether it's rolling back TO this tag, or rolling back JUST this tag.